core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -566,6 +568,7 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
 
     DefensiveBonus("Gives a defensive bonus of [relativeAmount]%", UniqueTarget.Improvement),
     ImprovementMaintenance("Costs [amount] gold per turn when in your territory", UniqueTarget.Improvement), // Unused
+    ImprovementAllMaintenance("Costs [amount] gold per turn", UniqueTarget.Improvement), // Unused
```

Comment:
A. We always allow for both old and new unique formats to work for a certain amount of time to allow modders to change existing unqiues
B. If you're replacing a unique, mark it with a @Deprecated so that it's both obvious in the docs and the unique can be automatically converted to the new one

Reply:
Now that I think of it...
If we're changing this unique anyway, why not allow *any* stats? "Costs [stats] per turn"

Reply:
Hookay, done